test/ProgressOnderwijsUtils.Tests/UtilsTest.cs

```diff
@@ -211,11 +211,32 @@ public void TryWithCleanup_CleansUpOnceWhenComputationFails()
 
             var ex = Assert.ThrowsAny<Exception>(() => value = Utils.TryWithCleanup(buggyComputation, cleanup));
 
-            PAssert.That(() => ex.Message == "1337" && ex.TargetSite == buggyComputation.Method);
+            PAssert.That(() => ex.Message == "Computation crashed (1337)" && ex.TargetSite == buggyComputation.Method);

```

Comment:
de exception message was veranderd en heb ik aangepast, maar de targetsite blijkbaar ook en ik kan zo niet inschatten of dat terecht is.

Reply:
Dat is vanwege de Action-to-Func conversie, dus ja, dat is terecht hier.  Ik zou de check gewoon weg doen.